crates/pallets/starknet/src/runtime_api.rs

```diff
@@ -18,6 +18,15 @@ use sp_runtime::DispatchError;
 use crate::types::{NonceWrapper, StateCommitments};
 use crate::StateTrie;
 
+#[derive(scale_codec::Encode, scale_codec::Decode, scale_info::TypeInfo)]
```

Comment:
No derive Error?

Don't seem to be needed. But is there a reason we would like it to be implemented?